src/components/Counter.js

```diff
@@ -20,12 +20,14 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
-                    +
-                </button>
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.count - 1 }>
                     -
                 </button>
+                
+                <button onClick={() => this.props.count + 1 }>
```

Comment:
If you are updating the state of the count in the reducer you shouldn't need to do that here as well